### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by LewWadoo**
This time it doesn't touch `eslint` dependency despite one of the commit messages.
- Upgrade eslint-config-airbnb
- Upgrade eslint-plugin-import, raw-loader, webpack-merge
- Upgrade glob
- Upgrade async
- Upgrade @babel/cli, @babel/core, @babel/preset-env
- Install eslint-plugin-react-hooks
- Update caniuse-lite
- Install eslint-plugin-react
- Install eslint-plugin-jsx-a11y

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,23 +38,26 @@
     "webpack"
   ],
   "dependencies": {
-    "async": "^3.2.3",
+    "async": "^3.2.4",
     "chalk": "^4.1.0",
-    "glob": "^7.1.6",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
```
Comment (justin808) [inline package.json:45]: devDependencies

**Comment by justin808** [comment]
@LewWadoo 
1. List how you tested the updates.
2. Is any new release justified?

**Review by justin808 — CHANGES_REQUESTED** [review]
and this needs updating

File: package.json (hunk 1 above)
Comment (LewWadoo) [reply package.json]: Done

**Comment by LewWadoo** [comment]
> @LewWadoo
> 
>     1. List how you tested the updates.
I relied on `yarn run test` e.g., `jest` tests. I used only this one option, so nothing to list.

>     2. Is any new release justified?
`webpack` should not be upgraded to a major release, because our documentation states that it supports **Webpack 4**. It's easier to justify minor and hotfix releases, but all my major upgrades were not justified. I just wanted to see how far I can go without breaking the tests.

**Comment by justin808** [comment]
Did CI run? I'm surprised that the code didn't need updates with the eslint changes.

**Review by Judahmeek — CHANGES_REQUESTED** [review]
LGTM, but would be best to wait on this until after https://github.com/shakacode/sass-resources-loader/pull/160 is merged.

**Comment by justin808** [comment]
@Judahmeek @LewWadoo what's needed to get CI to run on this branch? #160 is merged.

**Comment by justin808** [comment]
And do we need a new release?

**Comment by LewWadoo** [comment]
> And do we need a new release?

I think we do. So we don't need to merge it blindly.

**Comment by Judahmeek** [comment]
@LewWadoo why not just rebase & force push to this branch instead of creating a new PR?

**Pull request closed without merging**